tests/test_components/test_custom.py

```diff
@@ -72,7 +72,8 @@ def make_spatial_data(value=0, dx=0, unstructured=False, seed=None, uniform=Fals
         data = np.random.random((Nx, Ny, Nz)) + value
     arr = td.SpatialDataArray(data, coords=dict(x=X + dx, y=Y, z=Z))
     if unstructured:
-        return cartesian_to_unstructured(arr, seed=seed)
+        method = "direct" if uniform else "linear"
```

Comment:
Why will `linear` slightly decrease eps_inf?

Reply:
https://github.com/flexcompute/tidy3d/blob/pre/2.7/tests/utils.py#L225-L234

Reply:
I see. It's possible that users might generate data after going through lots of steps (e.g. interpolation)  and then come across this eps_inf<1 error? I guess it'll be confusing for users to figure it out.

On the other hand, I think we need to update backend too if we only warn for eps_inf < 1- fp_eps. So probably good to merge yours for now?

Reply:
Yeah, I imagine that could happen. It seems like we haven't had issues like that with cartesian custom medium, so it's probably ok to keep this check strict